politeiad/backend/gitbe/gitbe.go

```diff
@@ -2745,7 +2745,14 @@ func (g *gitBackEnd) Inventory(vettedCount, branchCount uint, includeFiles, allV
 			}
 		}
 	}
-
+	if vettedCount != 0 {
```

Comment:
This still reads all of the records into memory, which means you'll still going to have the issue of running out of memory.  You need to implement the `VettedCount` and `VettedStart` in the for loop that reads the records into memory.